orca/server_metrics.go

```diff
@@ -63,21 +67,24 @@ func (sm *ServerMetrics) toLoadReportProto() *v3orcapb.OrcaLoadReport {
 
 // merge merges o into sm, overwriting any values present in both.
```

Comment:
This isn't correct? You overwrite any values for keys in map unconditionally, but only overwrite sm's memory if it's set. I guess if an explicit -1 means unset 1:1 this is right. I just don't know all the invariants of this system/how this data is plumbed around.

Reply:
> You overwrite any values for keys in map unconditionally, but only overwrite sm's memory if it's set.

No?  I range over `o.<map field>` and add all those key/value pairs into `sm.<map field>`.  It's a merge, not a replacement, so the idea is to keep anything in `sm` that's not in `o`.